### PR TITLE
Package xarray-leaflet

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/xarray_leaflet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "xarray_leaflet" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "5f68b61633392404830c6abebe5f31733a87a25bb3eb5cb10d1b1b2491dec38f" %}
+{% set build = 0 %}
```
Comment (xhochy) [inline recipes/xarray_leaflet/meta.yaml:4]: Please only use Jinja for name and version
```suggestion
```

File: recipes/xarray_leaflet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - python
+    - xarray
+    - ipyleaflet
+    - pillow
+    - rasterio
+    - affine
+    - mercantile
+
+test:
```
Comment (xhochy) [inline recipes/xarray_leaflet/meta.yaml:32]: Always good to check the dependencies are fulfilled.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/xarray_leaflet/meta.yaml (hunk 1 above)
Comment (davidbrochart) [reply recipes/xarray_leaflet/meta.yaml]: Thanks, I didn't know.

File: recipes/xarray_leaflet/meta.yaml (hunk 2 above)
Comment (davidbrochart) [reply recipes/xarray_leaflet/meta.yaml:30]: Thanks!

File: recipes/xarray_leaflet/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipes/xarray_leaflet/meta.yaml]: These are just some tricks to ensure that the autotick-bot has an easier job ;)

**Comment by davidbrochart** [comment]
Thanks!

**Pull request merged**